src/Explorer/Panes/LoadQueryPanel/index.tsx

```diff
@@ -0,0 +1,147 @@
+import { useBoolean } from "@uifabric/react-hooks";
+import { IImageProps, Image, ImageFit, Stack, TextField } from "office-ui-fabric-react";
+import React, { FunctionComponent, useState } from "react";
+import folder_16x16 from "../../../../images/folder_16x16.svg";
+import { logError } from "../../../Common/Logger";
+import { userContext } from "../../../UserContext";
+import {
+  clearInProgressMessageWithId,
+  logConsoleError,
+  logConsoleInfo,
+  logConsoleMessage,
+  logConsoleProgress,
+} from "../../../Utils/NotificationConsoleUtils";
+import Explorer from "../../Explorer";
+import { ConsoleDataType } from "../../Menus/NotificationConsole/NotificationConsoleComponent";
+import QueryTab from "../../Tabs/QueryTab";
+import { Collection } from "..//../../Contracts/ViewModels";
+import { GenericRightPaneComponent, GenericRightPaneProps } from "../GenericRightPaneComponent";
+
+interface LoadQueryPanelProps {
+  explorer: Explorer;
+  closePanel: () => void;
+}
+
+export const LoadQueryPanel: FunctionComponent<LoadQueryPanelProps> = ({
```

Comment:
Can we get some unit test on this component? Maybe an enzyme snapshot?

Reply:
added snapshot.